src/blockdata/transaction.rs

```diff
@@ -1233,3 +1233,60 @@ mod tests {
     }
 }
 
+#[cfg(all(test, feature = "unstable"))]
+mod benches {
+    use super::Transaction;
+    use EmptyWrite;
+    use consensus::{deserialize, Encodable};
+    use hashes::hex::FromHex;
+    use test::{black_box, Bencher};
+
+    const SOME_TX: &'static str = "0100000001a15d57094aa7a21a28cb20b59aab8fc7d1149a3bdbcddba9c622e4f5f6a99ece010000006c493046022100f93bb0e7d8db7bd46e40132d1f8242026e045f03a0efe71bbb8e3f475e970d790221009337cd7f1f929f00cc6ff01f03729b069a7c21b59b1736ddfee5db5946c5da8c0121033b9b137ee87d5a812d6f506efdd37f0affa7ffc310711c06c7f3e097c9447c52ffffffff0100e1f505000000001976a9140389035a9225b3839e2bbf32d826a1e222031fd888ac00000000";
```

Comment:
Perhaps add a line of comment on this tx. Why was it picked? Is it mainnet? Is it large? ...